addons/website_sale/models/sale_order.py

```diff
@@ -471,3 +471,11 @@ def action_preview_sale_order(self):
             # URL should always be relative, safety check
             action['url'] = f'/@{action["url"]}'
         return action
+
+    def _get_website_sale_extra_values(self):
+        """
+        This method allows extending modules to provide additional values for the cart template.
+        :param order: The sale.order object of the current order
+        :return: A dictionary of additional values to be passed to the cart template
+        """
```

Comment:
```suggestion
        """ Hook to provide additional rendering values for the cart template.
        :return: additional values to be passed to the cart template
        :rtype: dict
        """
        self.ensure_one()
```
The docstring is not up to date with the latest signature ;)